Project.toml

```diff
@@ -24,8 +24,7 @@ StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-Arrow = "0.3, 0.4, 1"
-BlockArrays = "0.11, 0.12"
+BlockArrays = "0.11, 0.12, 0.13"
 DataAPI = "1.1, 1.2, 1.3"
```

Comment:
Should we add 1.4 for DataAPI (or just reduce the compat entry to "1") while making this change?

Reply:
Why not? :)